pkg/entities/entity_integration_test.go

```diff
@@ -141,7 +141,11 @@ func TestIntegrationGetEntity(t *testing.T) {
 	t.Parallel()
 
 	// GUID of Dummy App
-	entityGUID := common.EntityGUID(testhelpers.IntegrationTestApplicationEntityGUID)
+	// entityGUID := common.EntityGUID(testhelpers.IntegrationTestApplicationEntityGUID)
+
+	// Temporarily using the GUID of a different app
+	entityGUID := common.EntityGUID("MzgwNjUyNnxBUE18QVBQTElDQVRJT058NTUzNDQ4MjAy")
+
```

Comment:
`func TestIntegrationGetEntity(t *testing.T)`
- _temporarily_ changed the GUID of the entity being used by the test to fetch details of, which is a Python-based APM entity, `Dummy App Pro Max`
- accordingly, changed assert statements in the following lines of the same function to match the ID of the entity, language, etc.